### PR TITLE
Usf patch

### PR DESCRIPTION
**Pull request by cwbell-usf**
Updates to make the USF rule functional and add all known public-facing domains. Three sites remain to be resolved, working with USF IT to determine cause.

### CODE REVIEW

**Comment by pipboy96** [comment]
@cwbell-usf Thanks for contribution! May I ask if you are a USF employee? If yes, I recommend using HSTS preloading, which would make this ruleset unnecessary or at least make it much simpler.

**Comment by cwbell-usf** [comment]
I am, and though I can't quite go into why (opsec) we currently can't roll out hsts for most of the domains. Things move slow in an org this big, so this is a stopgap until we can enforce natively.

File: src/chrome/content/rules/University-of-South-Florida.xml [hunk 1]
```diff
@@ -1,68 +1,51 @@
...
-	<target host="vpn.usfsp.edu" />
+	<target host="www.cutr.usf.edu" />
+	
+	<target host="health.usf.edu" />
+	
 	<target host="www.usfsp.edu" />
 	<target host="sharemypc.usfsp.edu" />
+	
+	<target host="wusf.usf.edu" />
+	<target host="www.wusf.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/University-of-South-Florida.xml]: `wusfnews.wusf.usf.edu`

File: src/chrome/content/rules/University-of-South-Florida.xml [hunk 2]
```diff
@@ -1,68 +1,51 @@
...
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://usfsp.edu/ => https://usfsp.edu/: (51, "SSL: no alternative certificate subject name matches target host name 'usfsp.edu'")
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://usfsp.edu/ => https://usfsp.edu/: (51, "SSL: no alternative certificate subject name matches target host name 'usfsp.edu'")
-	For problematic rules, see University-of-South-Florida-mismatches.xml.
-
-
-	Nonfunctional usf.edu subdomains:
```
Comment (J0WI) [inline src/chrome/content/rules/University-of-South-Florida.xml]: Please take care that you're not removing any information that is still valid.

File: src/chrome/content/rules/University-of-South-Florida.xml (hunk 1 above)
Comment (cwbell-usf) [reply src/chrome/content/rules/University-of-South-Florida.xml]: That domain has been deprecated and will no longer be maintained

File: src/chrome/content/rules/University-of-South-Florida.xml (hunk 2 above)
Comment (cwbell-usf) [reply src/chrome/content/rules/University-of-South-Florida.xml]: I wasn't aware that I did? I removed rules that do/will not have valid SSL protection on them or that are no longer in production/do not exist/are not maintained. 

The usfsp.edu no SAN one in particular will not be resolved.

File: src/chrome/content/rules/University-of-South-Florida.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/University-of-South-Florida.xml]: Would be good to protect it as long as it's still up.

File: src/chrome/content/rules/University-of-South-Florida.xml (hunk 2 above)
Comment (J0WI) [reply src/chrome/content/rules/University-of-South-Florida.xml]: E.g. `system.usf.edu` is still a a non-functional target. We use to document them to make sure that we haven't missed them: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

Comment (cwbell-usf) [reply src/chrome/content/rules/University-of-South-Florida.xml]: Good point... Ok, I guess I'll work on documenting those and resub a new PR later.

**Comment by cwbell-usf** [comment]
I'll open a new one when I have more documentation.

**Pull request closed without merging**